README.md

```diff
@@ -74,6 +74,74 @@ Server:
  Experimental: false
 ```
 
+#### Mounting EBS volumes (hard drive storage)
+
+To make an Elastic Block Store (EBS) volume available to the databox, there a few steps you need to follow. These steps are defined in the amazon web services [documentation](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-using-volumes.html), and replicated in brief here.
```

Comment:
* How do you create the EBS volume?
* How are you planning to mount it?

I haven't checked the latest Terraform/Ansible code, but I assume you may want to create the EBS resource using Terraform and then instruct the running instance to use it through Ansible commands, is it right?

Reply:
Oh sorry I completely missed the fact that you had changed the Terraform script to add the creation of the EBS volume.

Yes, I think we can use Ansible to script those commands. Do you want me to pick this task?

Cheers

Reply:
yeah, that would be great 👍

Reply:
the script was already creating the volume though - i just changed the script to change teh volume size.

Reply:
merging this then